packages/core/src/geography/index.ts

```diff
@@ -1,8 +1,10 @@
 export * from './country-responses';
+export * from './united-state-codes';
 
 export { default as CountryActionCreator } from './country-action-creator';
 export { default as CountryRequestSender } from './country-request-sender';
-export { default as Country } from './country';
+export { default as Country, Region, GetCountryResponse, UnitedStatesCode } from './country';
```

Comment:
Should we remove GetCountryResponse import?

Reply:
removed